ee/codegen/src/__test__/helpers/node-data-factory-builder.ts

```diff
@@ -0,0 +1,29 @@
+import { NodeAttribute, NodePort } from "src/types/vellum";
+
+export class NodeDataFactoryBuilder<T> {
```

Comment:
[nit] Ideally `T` is a `T extends WorkflowNodeType` or whatever the latter is called in our repo